packages/react-native-sdk/src/providers/StreamCall/DeviceStats.tsx

```diff
@@ -7,8 +7,10 @@ import {
 } from '@stream-io/video-client';
 import { NativeEventEmitter, NativeModules, Platform } from 'react-native';
 
-const eventEmitter = NativeModules?.StreamVideoReactNative
-  ? new NativeEventEmitter(NativeModules?.StreamVideoReactNative)
+const StreamVideoReactNative = NativeModules?.StreamVideoReactNative;
```

Comment:
```suggestion
const StreamVideoReactNative = NativeModules.StreamVideoReactNative;
```

its our sdk, the module will always be present